Ram classes screen and Filtering Screen

ramzd18 opened the pull request:
Built the Basic UI for the Classes and Filtering Screen. The data is still hardcoded at the moment so I will have to in the future create a view model and actually apply the filtering to real data.

ramzd18 committed:
Building Initial Classes Screen UI

Created initial UI for the Classes Screen and some logic for the calendar.

ramzd18 committed:
Adding Partial Class Filter UI

Built parts of the filtering UI for the classes screen.

ramzd18 committed:
Dropdowns on Filtering Screen

Added dropdown card components for filtering screen

ramzd18 committed:
Touches to Class Screen Ui

Fixed Button and some Padding issues

Corfish123 commented:
I don't seem to be able to see the UI components you made

Corfish123 commented on app/src/main/java/com/cornellappdev/uplift/ui/screens/Filtering Screen.kt:
```diff
@@ -0,0 +1,422 @@
+import com.cornellappdev.uplift.util.*
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
```
comment these compossables

Corfish123 commented on app/src/main/java/com/cornellappdev/uplift/ui/screens/ClassScreen.kt:
```diff
@@ -0,0 +1,130 @@
+import com.himanshoe.kalendar.Kalendar
+import com.himanshoe.kalendar.model.KalendarType
+
+class dateInstance(var date: Int, var day1:String) {
```
comment all of this

Corfish123 commented on app/src/main/java/com/cornellappdev/uplift/ui/screens/Filtering Screen.kt:
```diff
@@ -0,0 +1,422 @@
+        }
+    }
+}
+@Composable
```
add comments to everything

ramzd18 committed:
Bug Touchups and addition of viewModel files

Fixed issues with coloring, boldness and added comments. Added two viewModel files for classes screen and filtering screens.

ramzd18 committed:
Fixed TopBar Padding

Fixed padding issues for Topbar and text values in Fitness Center Row

Corfish123 reviewed (approved):
Great work
